instrumentation/racecar/lib/opentelemetry/instrumentation/racecar/process_message_subscriber.rb

```diff
@@ -4,14 +4,21 @@ module OpenTelemetry
   module Instrumentation
     # This class contains the ASN subsciber that instruments message processing
     class ProcessMessageSubscriber
+      GETTER = if Gem::Version.new(::Rdkafka::VERSION) >= Gem::Version.new('0.13.0')
```

Comment:
I would _prefer_ to see this evaluation occur in `install` process since we do other gem version compatibility checks but 🤷🏼‍♂️ it may make it more difficult to understand. WDYT @kaylareopelle ?

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/b26b765b91b789bce25d47743326d3cc16b6feb7/instrumentation/racecar/lib/opentelemetry/instrumentation/racecar/instrumentation.rb#L36